Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7
+FROM python:3.10-slim-bullseye
```

Comment:
I don't have a lot of experience running the `slim`. I think we will be fine since we should only be running python based applications.